lib/processors/newsprocessor.py

```diff
@@ -0,0 +1,147 @@
+from tornado import gen
+from .lib.utils import process_api_handler
+
+import re
```

Comment:
Yeah, I put them in there because I was going to try to do some more complex stuff and haven't had the time to clean up the processors to a more polished state. I figured that was the kind of things I would do in branches after this basic functionality was merged.